lib/Makefile.Watcom

```diff
@@ -272,4 +272,3 @@ $(RESOURCE): libcurl.rc
 
 .c{$(OBJ_STAT)}.obj:
 	$(CC) $(CFLAGS) -DCURL_STATICLIB $[@ -fo=$^@
-	
```

Comment:
Can we be certain that all makefiles which aren't regularly tested will accept this, or can there be weird incantations of make programs on platforms which want a blank line at the end? To be clear, I don't think there is much of a risk but I at the same time don't know (and having worked on a project vendoring pmake in the distant past I'm no stranger to odd `make`s).

Reply:
I haven't yet seen any app that would require duplicate newlines at the end of text files. GNU Make for certain doesn't need it. I haven't used Watcom's make in particular, but the pre-patch state is a single EOL in `src/Makefile.watcom` and a stray tab then a single EOL in `lib/Makefile.watcom` at the end of the files — but no double newlines in this case. The stray tab is most likely a one-time typo. Rest of the files involved is mostly sources, GNU Make files, text files (for humans mostly). Amiga make files look fairly GNU Make-like, but none had double ending newlines, only some tabs. No other make files were modified. One `.spec` files is, but it should be fine.

Any extra eyeballs on this are appreciated of course! I may have missed something.